### PR TITLE
Update versions in Travis/AppVeyor configuration

### PR DESCRIPTION
**Pull request by sdroege**
And also run tests with default features for all the crates.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
     packages:
     - libgtk-3-dev
 script:
   - rustc --version
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ]; then
+  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.14" ]; then
     make regen_check;
     fi
+  - for crate in "atk-sys gdk-pixbuf-sys gdk-sys gio-sys glib-sys gobject-sys gtk-sys pangocairo-sys pango-sys"; do
+    cd $crate && cargo test && cd ..
```
Comment (EPashkin) [inline .travis.yml]: Better don't do this now, it take too much time and fails.

**Comment by EPashkin** [comment]
Your gir repo has changes "Generated by gir (https://github.com/gtk-rs/gir @ 4a198d9+)"

File: .travis.yml (hunk 1 above)
Comment (sdroege) [reply .travis.yml]: Ah so nobody actually fixed the test since they were merged... disappointing :( Then I disable that again.

**Comment by sdroege** [comment]
> Your gir repo has changes "Generated by gir (https://github.com/gtk-rs/gir @ 4a198d9+)"

What do you mean?

**Comment by sdroege** [comment]
I see. I can't reproduce this here unfortunately. Where does that come from?

**Comment by sdroege** [comment]
I mean, `make regen_check` finishes without any diff at the end here.

**Comment by EPashkin** [comment]
Strange error, don't understand it reason, restarted job just in case

**Comment by EPashkin** [comment]
Found problem, new cargo add comment in lock file.
@sdroege please update gir to 27232165a279c49da2e740e3d3eee2b014ba5516
```
+++ b/Cargo.lock
@@ -1,3 +1,5 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
 [[package]]
 name = "aho-corasick"
```

**Comment by sdroege** [comment]
Done, thanks for finding the reason :)

**Comment by sdroege** [comment]
@GuillaumeGomez all green

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**